partridge/gtfs.py

```diff
@@ -241,4 +241,4 @@ def trip_counts_by_date(self):
 # No pruning or type coercion
 class raw_feed(feed):
     def __init__(self, path):
-        super().__init__(path, config=nx.DiGraph())
+        super(raw_feed, self).__init__(path, config=nx.DiGraph())
```

Comment:
Is this meant to be `super(feed, self)`?

Reply:
Heh, bit late with my review. Sorry! 😳

Reply:
I think this is right. See the examples in https://docs.python.org/2/library/functions.html#super.

It looks like the reference is to the subclass.

Reply:
Oh, I'm sorry, you're absolutely right. I've forgotten how to use that statement. 👍